tests/member/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -1,5 +1,17 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`conditional.js 1`] = `
+(valid
+  ? helper.responseBody(this.currentUser)
+  : helper.responseBody(this.defaultUser))
+.prop;
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+(valid
+  ? helper.responseBody(this.currentUser)
+  : helper.responseBody(this.defaultUser)).prop;
```

Comment:
I understand why it's happening, but this kinda feels like a regression on the changes in https://github.com/prettier/prettier/pull/2784. I think I'm okay with it, though.

Reply:
Actually, that's my mistake I forgot to update this test :flushed: